<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="quoteLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>QUOTE HERE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AnswerButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Answer</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>371</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ForwardButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HandleButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Mark as done</string>
    </property>
   </widget>
   <widget class="QListWidget" name="IncomingCall">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>371</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pending Calls:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>History Log:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearAllDoneButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear all done status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearAllCancelButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear all cancelled status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HandleButton2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Mark as cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
